tests/integration/session/session.test.js

```diff
@@ -21,16 +20,15 @@ describe('Module Athentication', () => {
 
         }
 
-        const response = await AuthenticateRepository.signin(data)
+        const response = await SessionRepository.signin(data)
 
         expect(response).toEqual(401)
 
     })
 
     it('should return jwt token when authenticated', async () => {
```

Comment:
need to describe more and better the tests ex:

context  'when need authenticate'
   context 'when user has ben logged'
       context 'When user has a valid token
            it 'should be return a object user
        context 'When user has a invalid token
             it 'should be return a invalid token error
         context 'When user don't have authorization
             it 'should be return 401